src/parser/hunter/shared/modules/talents/AMurderOfCrows.js

```diff
@@ -17,26 +18,61 @@ import StatisticListBoxItem from 'interface/others/StatisticListBoxItem';
 const CROWS_TICK_RATE = 1000;
 const MS_BUFFER = 100;
 const CROWS_DURATION = 15000;
+const debug = false;
 
 class AMurderOfCrows extends Analyzer {
 
   static dependencies = {
     spellUsable: SpellUsable,
+    abilities: Abilities,
   };
 
   bonusDamage = 0;
   casts = 0;
   applicationTimestamp = null;
   lastDamageTick = null;
   crowsEndingTimestamp = null;
+  maxCasts = 0;
 
   constructor(...args) {
     super(...args);
     this.active = this.selectedCombatant.hasTalent(SPELLS.A_MURDER_OF_CROWS_TALENT.id);
+    if (!this.active) {
+      return;
+    }
+    this.abilities.add({
+      spell: SPELLS.A_MURDER_OF_CROWS_TALENT,
+      category: Abilities.SPELL_CATEGORIES.ROTATIONAL,
+      cooldown: 60,
+      gcd: {
+        base: 1500,
+      },
+      castEfficiency: {
+        suggestion: true,
+        recommendedEfficiency: 0.85,
+        maxCasts: () => this.maxCasts,
+      },
+      ...this.constructor.extraAbilityInfo,
+    });
+  }
+
+  checkForReset(event) {
+    // Checks if we've had atleast 1 damage tick of the currently applied crows, and checks that crows is in fact on cooldown.
+    if (this.lastDamageTick && this.spellUsable.isOnCooldown(SPELLS.A_MURDER_OF_CROWS_TALENT.id)
+      // Checks whether the current damage event happened while the time passed since crows application is less than the crows duration
+      && this.applicationTimestamp && event.timestamp < this.crowsEndingTimestamp
+      // Checks to see if more than 1 second has passed since last tick
+      && event.timestamp > this.lastDamageTick + CROWS_TICK_RATE + MS_BUFFER) {
```

Comment:
Not quite how reliable this is if the ticks are scaling with haste (esp when thinking about negative haste like we had in NH).
But it'll fix more than it breaks